pom.xml

```diff
@@ -59,47 +59,54 @@
       <artifactId>gson</artifactId>
       <version>2.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>me.moocar</groupId>
+      <artifactId>socket-encoder-appender</artifactId>
+      <version>0.1beta1</version>
+    </dependency>
```

Comment:
The socket-encoder-appender is technically not required in non-test scope. I had envisaged users including that dependency only if they need TCP functionality. So it can have the test scope added and moved back down.

But let me know if you think that's being too pure. It only adds 9kb to the overall jar so may be worth including